stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -1257,47 +1255,6 @@ fileprivate func setAttrShapeList(
   }
 }
 
-@usableFromInline
```

Comment:
I think we will still need the `_ThreadLocalState`. Adding @pschuh as he wrote most of this code.

Reply:
Good to know that we need this to be thread-safe. Will fix.